### PR TITLE
Added side bar with routes to helpful feed

### PR DESCRIPTION
**Pull request by JimmyG37**

### CODE REVIEW

File: client/src/components/Helpful/HelpfulFeed.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const SideMenu = styled.div`
+ float: left;
+`;
+
+const HelpfulFeed = () => {
+  const tasks = useSelector((store) => store.tasksReducer.tasks);
+
+
+
+  const placeholder = 'NexDoors Most Helpful';
```
Comment (raphaelspies) [inline client/src/components/Helpful/HelpfulFeed.jsx]: needs an apostrophe before the 's' in NexDoors

File: client/src/components/MainFeed/index.jsx [hunk 2]
```diff
@@ -16,18 +19,40 @@ const MapPlaceholder = styled.div`
...
-    <Tasks />
-    <MapPlaceholder />
-  </Grid>
-);
+const MainFeed = () => {
+  const tasks = useSelector((store) => store.tasksReducer.tasks);
+  // const dispatch = useDispatch();
+  const placeholder = 'placeholder';
+
+  // const getTasks = () => {
```
Comment (raphaelspies) [inline client/src/components/MainFeed/index.jsx:27]: delete unused comments

File: client/src/components/Sidebar.jsx [hunk 3]
```diff
@@ -20,39 +21,53 @@ const SidebarContainer = styled.div`
...
+        <ListItemText primary="Open Tasks" />
+      </ListItem>
+      <ListItem button>
+        <ListItemIcon>
+          <Layers />
+        </ListItemIcon>
+        <ListItemText primary="Active Tasks" />
+      </ListItem>
+      <ListItem button onClick={handleHelpfulFeed}>
+        <ListItemText primary="NexDoors Most Helpful" />
```
Comment (raphaelspies) [inline client/src/components/Sidebar.jsx]: apostrophe before s

**Review by raphaelspies — CHANGES_REQUESTED** [review]
Looks good! Got a couple minor revisions for you to make.

File: client/src/components/MainFeed/index.jsx (hunk 2 above)
Comment (JimmyG37) [reply client/src/components/MainFeed/index.jsx:27]: still need it

**Review by raelyncs — APPROVED** [review]
Nice work!

**Review by raphaelspies — APPROVED** [review]

**Pull request merged**